lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -582,10 +582,13 @@ module.exports = class MethodMappings {
 
           return value;
         },
-
+        
         async clearElementValue(webElementOrId) {
           const element = this.getWebElement(webElementOrId);
           await element.clear();
+          var value = await element.getAttribute('value');
```

Comment:
Use `const`/`let` instead of `var`. Also, use `getProperty` method instead of `getAttribute`.

Reply:
Also enclose the code you added in `try-catch` block so that any errors while fetching the value doesn't go back to the user.

Reply:
Can you format the code as well? You can use eslint to do that.

Reply:
Hi!
Sure 
Actually I was unsure about linting that's why I skipped it since sometimes it's different for Linux and Windows